README.md

```diff
@@ -619,15 +619,17 @@ Executes JavaScript code from `commands` on current page, uses found element as
 To place element as argument to code, use "@Element@" reference.
 
 
-### PrintBrowserLogs
+### GetBrowserLogs
 ```
 #action
-PrintBrowserLogs
+GetBrowserLogs
 ```
 
-Prints current browser logs in RemoteHand logs. Useful to check page on JavaScript errors.
+Returns string that contains current browser logs. Useful to check page on JavaScript errors.

```

Comment:
Replace "Useful to check page on JavaScript errors." with "Useful to check page for JavaScript errors."